dart/welcome_email/pubspec.yaml

```diff
@@ -0,0 +1,10 @@
+name: functions_demo_dart_welcome_email
+description: ""
+version: 1.0.0
+
+environment:
+  sdk: '>=2.12.0 <3.0.0'
+
+dependencies:
+  dart_appwrite: ^0.6.1
```

Comment:
Is this required?

Reply:
No, it isn't it I don't think

Reply:
Yes, since we require the http library however I will remove the appwrite dependency